.github/actions/create-issue/action.yaml

```
@@ -13,13 +13,15 @@ inputs:
 runs:
   using: "composite"
   steps:
-    - name: Checkout code
+    - name: Checkout this repo
```

```suggestion
    - name: Checkout this sigstore/sigstore-probers
```

nit: just to make more clear